modules/ROOT/pages/scalability/sharded-property-databases/data-ingestion.adoc

```diff
@@ -0,0 +1,208 @@
+:page-role: new-2025.10 enterprise-edition not-on-aura
+:description: Methods for ingesting data into a sharded property database.
+:keywords: sharded property database, data ingestion, import, neo4j-admin, LOAD CSV, CREATE DATABASE
+= Data ingestion
+
+There are several ways to load data into a sharded property database:
+
+* Creating a sharded property database from delimited files using `neo4j-admin database import`.
+* Creating a sharded property database by splitting an existing Neo4j database.
+* Loading data transactionally into an existing sharded property database.
+* Incrementally updating an existing sharded property database.
+
+== Offline ingestion
+
+You can use the offline ingestion methods to initially import data into a sharded property database.
+This is useful when you want to import in bulk before starting your application, for incremental imports later on, or for splitting an existing database into shards.
+
+=== Initial import from delimited files
+
+You can use the xref:import.adoc#import-tool-full[`neo4j-admin database import full`] command to import data from delimited files into a sharded property database as in a standard Neo4j database.
+This is particularly useful for large datasets that you want to import in bulk before starting your application or for incremental imports later on.
+You can specify the `--property-shard-count` option to define the number of property shards you want to create.
+This will help distribute the data across multiple servers in a Neo4j cluster.
+
+[NOTE]
+====
+If you are creating the property shards on a self-managed server, the server that executes the `neo4j-admin database import` command must have sufficient storage space available for all of the property shards that will be created.
+====
+
+The following example shows how to import a set of CSV files,  dump them to S3, and then create a database using those seeds in S3.
+
+. Using the `neo4j-admin database import` tool, import data into the `foo-sharded` database, creating one graph shard and three property shards.
+If the process is running on the same server as another Neo4j DBMS process, the latter must be stopped.
+The `-–target-location` and `-–target-format` options will take the outputs of the import, turn them into backups, and upload them to a location ready to be seeded from.
+The backup will be uncompressed, which is the quickest way to perform a backup.
++
+[source, shell]
+----
+neo4j-admin database import full foo-sharded --nodes=nodes.csv --nodes=movies.csv --relationships=relationships.csv --input-type=csv --property-shard-count=3 --schema=schema.cypher --target-location=s3://bucket/folder/ --target-format=backup
+----
+
+. Create the database foo-sharded as a sharded property database by seeding it from your dumps in the AWS S3 bucket:
```

Comment:
In the previous paragraph, we mention that the `target-format` option will create an uncompressed back.
This paragraph mentions a dump file. It should be consistent verbiage.